src/app/views/seminars/seminars.component.html

```diff
@@ -0,0 +1,75 @@
+<div id="GuestSeminars" class="p-4 row d-flex justify-content-center col-12 pt-0" *ngIf="selectedTab === 'GuestSeminars'">
+  <ng-container *ngFor="let item of seminars.ICWaR_Guest_Seminar_Series; index as index">
+    <div class="card-bg shadow border border-1 mb-2 card col-12 col-md-10 border-dark p-3">
+      <div class="card-text row">
+        <div class="col-md-9 col-12 mb-2">
+          <h4><b>{{item.DateTime}}</b></h4>
+          <h3 class="text mb-3"><b>{{item.Title}}</b></h3>
+          <img *ngIf="item.imgSrc" class="mb-3 d-md-none float-end img-fluid" style="max-width: 125px;" [src]="item.imgSrc" />
+          <h5><b>Speaker: </b>{{item.speakerName}}</h5>
+          <h5><b>Platform: </b>{{item.Platform}} <a *ngIf="item.link" [href]="item.link" target="_blank">To Join/Watch</a></h5>
+          <br>
+        </div>
+        <div class="col-md-3 col-12 d-none d-md-block">
+          <img *ngIf="item.imgSrc" class="my-2 img-fluid" style="width:99%; float: right; max-width: 150px;" [src]="item.imgSrc" />
+        </div>
+        <div class="col-12">
+          <div id="{{index}}" class="row py-2 col-12 collapse">
+            <h5><b>Abstract:</b></h5>
+            <h6 class="text mb-3"><b></b>{{item.Abstract}}</h6>
+            <h5><b>About the Speaker:</b></h5>
+            <h6 class="text">{{item.AboutSpeaker}}</h6>
+          </div>
+          <div class="col-12 d-flex">
+            <button class="m-1 p-1 btn btn-text-expand collapsed ms-auto" type="button" data-bs-toggle="collapse" [attr.data-bs-target]="'#'+index" aria-expanded="false" [attr.aria-controls]="index"></button>
+          </div>
+          <hr style="height: 1px; background-color: black;">
+        </div>
+      </div>
+    </div>
+  </ng-container>
+</div>
+
+<!-- Community Seminars Section -->
+<div id="CommunitySeminars" class="p-4 row d-flex justify-content-center col-12 pt-0" *ngIf="selectedTab === 'CommunitySeminars'">
+  <ng-container *ngFor="let item of seminars.ICWaR_Community_Seminar_Series; index as index">
+    <div class="card-bg shadow border border-1 mb-2 card col-12 col-md-10 border-dark p-3">
+      <div class="card-text">
+        <h4><b>{{item.DateTime}}</b></h4>
+        <h3 class="text"><b>{{item.Title}}</b></h3>
+        <h5 class="col-md-9 col-12 mb-md-0"><b>Speaker:</b> {{item.speakerName}}</h5>
+        <div id={{index}} class="row py-4 col-12 collapse">
+          <h5><b>Abstract:</b></h5>
+          <h6 class="text col-12" >{{item.Abstract}}</h6> <br>
+        </div>
+        <div class="col-12 d-flex">
+          <button class="m-1 p-1 btn btn-text-expand collapsed ms-auto" type="button" data-bs-toggle="collapse" [attr.data-bs-target]="'#'+index" aria-expanded="false" [attr.aria-controls]="index"></button>
+        </div>
+        <hr style="height: 1px; background-color: black;">
+      </div>
+    </div>
+  </ng-container>
+</div>
+
+
+<button class="scroll-to-top" #scrollToTopButton (click)="scrollToTop()">↑</button>
```

Comment:
↑ Remove this this arrow icon. use class = "scroll-to-top fa fa-arrow".
Do it throughout the site